### ISSUE TITLE
The documentation doesn't scrollTo

### ISSUE THREAD

**Issue by Helveg** [nest/nest-simulator#2494]
**Describe the flaw**
Clicking the section items in the navbar doesn't navigate to the sections. My scroll finger hurts. Even navigating to their permalinks doesn't work:

https://nest-simulator.readthedocs.io/en/v3.3/ref_material/pynest_apis.html#module-nest.lib.hl_api_models

Update 1: Ah, seems like it also doesn't do it on other pages: https://nest-simulator.readthedocs.io/en/v3.3/getting_started.html#want-to-know-more.

Update 2: Clicking the same item in the navbar 2 times unselects the item visually, but triggers the navigation!

Update 3: Clicking any of the items in a "On this page you'll find" navbar moves the page about 10 pixels closer to its target per click on a fresh page, but works with double clicks after using the double click trick on the left navbar, until you refresh. https://nest-simulator.readthedocs.io/en/v3.3/guides/connection_management.html#pairwise-bernoulli

**Comment by Helveg** [comment]
Ah, seems like it also doesn't do it on other pages: https://nest-simulator.readthedocs.io/en/v3.3/getting_started.html#want-to-know-more

**Comment by jessica-mitchell** [comment]
@Helveg the scroll to feature works for me, what browser are you using?

**Comment by Helveg** [comment]
On Chrome, both on my laptop and mobile device.

**Comment by jessica-mitchell** [comment]
Hmm, I don't have this issue on Chrome or Firefox on my laptop, but Chrome on mobile doesnt work for me either. 
Edit: Ok interesting, I just did an update to Chrome and now the scrollto feature stops working. Unfortunately, we can't make changes to the source files for a pre-existing release.


I also checked latest (master) on mobile/laptop Chrome and the scrollto feature works with this version.
